src/components/cms/SkillCardStyle.js

```diff
@@ -49,6 +49,11 @@ export const TotalRating = styled.span`
   padding-left: 5px;
   color: #108ee9;
 `;
+export const AverageRating = styled.span`
```

Comment:
Please try to write some common code instead of adding duplicate code.
Both `TotalRating` and `AverageRating` have same set of code